src/components/CollaboratorModal/submodals/MainSubmodal.tsx

```diff
@@ -19,32 +18,41 @@ import {
   FormLabel,
 } from "@opengovsg/design-system-react"
 import { useCollaboratorModalContext } from "components/CollaboratorModal/CollaboratorModalContext"
+import { useFormContext } from "react-hook-form"
 import { BiTrash } from "react-icons/bi"
 
-import { CollaboratorModalState } from "../constants"
+import { useLoginContext } from "contexts/LoginContext"
+
+import { CollaboratorError } from "types/collaborators"
+import { DEFAULT_RETRY_MSG } from "utils"
+
+import {
+  CollaboratorModalState,
+  ACK_REQUIRED_ERROR_MESSAGE,
+} from "../constants"
 
 const LAST_LOGGED_IN_THRESHOLD_IN_DAYS = 60
```

Comment:
nit: shall we shift this `LAST_LOGGED_IN_THRESHOLD_IN_DAYS = 60` into `CollaboratorModal/constants.ts`?